src/include/optimizer/stats/column_stats_collector.h

```diff
@@ -0,0 +1,88 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// column_stats_collector.h
+//
+// Identification: src/include/optimizer/stats/column_stats_collector.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <vector>
+#include <functional>
+#include <libcount/hll.h>
+
+#include "type/types.h"
+#include "optimizer/stats/count_min_sketch.h"
+#include "optimizer/stats/top_k_elements.h"
+#include "optimizer/stats/histogram.h"
+#include "optimizer/stats/hyperloglog.h"
+
+namespace peloton {
+namespace optimizer {
+
+//===--------------------------------------------------------------------===//
+// ColumnStatsCollector
+//===--------------------------------------------------------------------===//
+class ColumnStatsCollector {
+ public:
+  /* Default parameters for probabilistic stats collector */
+  int hll_precision = 8;
+  double cmsketch_eps = 0.01;
+  double cmsketch_gamma = 0.01;
+  uint8_t max_bins = 100;
+  uint8_t top_k = 10;
```

Comment:
Giving those variables public access is for quick access of approximation algorithm's precisions and error bound outside of this class. I guess giving them getter and setter also works but make them public fits more of the design purpose (right now they are not used but we still want to expose those information for public access)